Homework 5-6

```diff
@@ -0,0 +1,98 @@
+HOMEWORK WEEK 5-6 
+(handout for students) 
+TASK 1 (Agile Techniques) 
+Question 1
+Complete definitions for Scrum related key terminology provided below. 
+SCRUM CEREMONIES 
+·  Product backlog refinement is a type of meeting held toward the end of the sprint to check that the backlog has been achieved before the next one
+•	Sprint planning, this is when you prioritise the tasks in a sprint and plan the order they are going to be actioned
+•	  Daily scrum , this is held at the start of the day to see the progress in a sprint held by a scrum master.
+•	  Sprint review, this is where you review the work produced by the sprint and whether further additions need to be made on backlog tasks
+•	  Sprint retrospective, this is after the sprint where you discuss what was useful/ not useful to apply to the next sprints in the future. 
```

Comment:
What is useful or not useful in what context?

A better definition/explanation of this is it is the scrum meeting/ceremony where the scrum team reflects on their previous sprint, giving feedback and finding ways on how to improve as a team and for future sprints. Questions such as: what went well, what did not and what can be improved are asked.